helix-core/src/test/java/org/apache/helix/integration/TestAlertingRebalancerFailure.java

```diff
@@ -164,6 +164,7 @@ public void testTagSetIncorrect() throws Exception {
         BuiltInStateModelDefinitions.MasterSlave.name(), RebalanceMode.FULL_AUTO.name());
     ZkHelixClusterVerifier verifier = new BestPossibleExternalViewVerifier.Builder(CLUSTER_NAME)
         .setZkAddr(ZK_ADDR).setResources(new HashSet<>(Collections.singleton(testDb))).build();
+    _gSetupTool.getClusterManagementTool().rebalance(CLUSTER_NAME, testDb, 3);
```

Comment:
3 -> _replica, please.

Reply:
@alirezazamani 
I see that this was marked as resolved but still see 3, not _replica. Has this PR been updated? I see your comment stating that this PR is ready to be merged. Is there something amiss here?

Reply:
@narendly 
We discussed offline and since in other parts of the test "3" has been used instead of _replica, we decided to keep the PR simple and keep it as it is.

Reply:
@alirezazamani Makes sense. Could you please make sure that kind of stuff is documented clearly in the PR next time? :)

Reply:
@alirezazamani 

Also, I think that your PR description could use some improvement. It doesn't give me any more information than the title, and it doesn't describe why it was unstable and how you were able to stabilize it. Please revise it, and please ping me again to get this merged into master.

Reply:
@narendly I updated the description.